src/main/resources/migrations.xml

```diff
@@ -180,6 +180,19 @@
 			<column name="createdAt" type="datetime" />
 			<column name="updatedAt" type="datetime" />
 		</createTable>
+
+		<createTable tableName="attachments">
+			<column name="id" type="int" autoIncrement="true" >
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="fileName" type="varchar(500)" />
+			<column name="mimeType" type="varchar(500)" />
+			<column name="content" type="varbinary(5000)" />
+			<column name="description" type="varchar(5000)" />
+			<column name="classificationUuid" type="varchar(500)" />
```

Comment:
All `varchar(5…)` should probably be `${long_text_type}`.
And `varbinary(5000)` (which is quite short) should be `${blob_type}`.
Also as customary, most tables should have a `createdAt` and `updatedAt` column.